test/Serilog.Sinks.RollingFile.Tests/RollingFileLoggerConfigurationExtensionsTests.cs

```diff
@@ -0,0 +1,16 @@
+﻿using System;
+using Xunit;
+
+namespace Serilog.Tests
+{
+    public class RollingFileLoggerConfigurationExtensionsTests
+    {
+        [Fact]
+        public void BuffferingIsNotAvailableWhenSharingEnabled()
+        {
+            Assert.Throws<ArgumentException>(() => 
+                new LoggerConfiguration()   
+                    .WriteTo.RollingFile("logs", buffered: true, shared: true));
+        }
+    }
+}
```

Comment:
Tests are admittedly a bit light, it's fair to assume the `SharedFileSink` does the heavy-lifting and is tested in its own repository, but some more coverage would be nice to add here at some point too.